packages/repack/src/modules/WebpackHMRClient.ts

```diff
@@ -128,7 +128,8 @@ class HMRClient {
       });
 
       if (!this.upToDate()) {
-        this.checkUpdates(update);
+        // TODO - should we return here?
+        void this.checkUpdates(update);
```

Comment:
this should probably be handled in separate PR - as this would cause a change in behaviour, needs to be checked whether returning here would still result in correct behaviour